Cargo.toml

```diff
@@ -8,13 +8,13 @@ edition = "2018"
 
 [dependencies]
 
-acvm = { git = "https://github.com/noir-lang/noir", rev = "cc5ee63072e09779bebd7e7dd054ae16be307d7f" } 
-arkworks_backend = { git = "https://github.com/noir-lang/arkworks_backend", rev = "ee6d468f5cc635dc2589965b6bf16cd2a839d7d3" }
+acvm = "0.3.1"
+arkworks_backend = { git = "https://github.com/noir-lang/arkworks_backend", rev = "2f3f0db182004d5c01008c741bf519fe6798e24d" }
 
 [features]
-# not specifying a default sometimes gives an error that "extern location for acvm does not exist" on some imports locally, 
+# not specifying a default sometimes gives an error that "extern location for acvm does not exist" on some imports locally,
 # but it still builds when specifying `marlin_arkworks_backend/[field name]` in the Cargo.toml inside the nargo crate
 # uncomment the default line to remove extern location errors, it is most likely a coloring error
-# default = ["bls12_381"] 
-bn254 = ["arkworks_backend/bn254"]
-bls12_381 = ["arkworks_backend/bls12_381"]
+# default = ["bls12_381"]
+bn254 = ["acvm/bn254", "arkworks_backend/bn254"]
```

Comment:
Would `acvm/bn254` be getting set in `arkworks_backend`?

Reply:
I've removed this, but I think that this circular feature flag stuff should be removed in the future.

Reply:
Sounds good, it would be nice to discuss this in its own issue as I'm curious to hear your thoughts. I think it is nice to be able to just specify `bn254` inside `nargo` with a feature flag and then we propagate the feature down to acvm which exposes the field.